packages/plugins/rubygem-openscap_parser/rubygem-openscap_parser.spec

```
@@ -65,11 +76,12 @@ find %{buildroot}%{gem_instdir}/bin -type f | xargs chmod a+x
 
 %files
 %dir %{gem_instdir}
-%exclude %{_bindir}/console
-%exclude %{_bindir}/setup
+%{_bindir}/console
+%{_bindir}/setup
 %exclude %{gem_instdir}/.gitignore
 %exclude %{gem_instdir}/.travis.yml
-%exclude %{gem_instdir}/Dockerfile
+%{gem_instdir}/CODE_OF_CONDUCT.md
```

We should handle this in our templates. Was probably also fine in the `-doc` package.
```suggestion
%doc %{gem_instdir}/CODE_OF_CONDUCT.md
```